src/main/java/com/lgtm/easymoney/repositories/AccountRepository.java

```diff
@@ -1,10 +1,11 @@
 package com.lgtm.easymoney.repositories;
 
 import com.lgtm.easymoney.models.Account;
-import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
```

Comment:
Don't change the import order, otherwise checkstyle complaims

Reply:
is this fixed? @KenXiong123